File: drift/src/main/java/drift/com/drift/helpers/StatusBarColorizer.java
package drift.com.drift.helpers;

import android.graphics.drawable.ColorDrawable;
import android.os.Build;
import androidx.appcompat.app.AppCompatActivity;


public class StatusBarColorizer {

    public static void setActivityColor(AppCompatActivity activity) {

        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
            activity.getWindow().setStatusBarColor(ColorHelper.getBackgroundColorLightened());
        }

        if (activity.getSupportActionBar() != null) {
            activity.getSupportActionBar().setBackgroundDrawable(new ColorDrawable(ColorHelper.getBackgroundColor()));
        }

    }
}

File: drift/src/main/java/drift/com/drift/helpers/StatusBarColorizer.kt
package drift.com.drift.helpers

import android.graphics.drawable.ColorDrawable
import android.os.Build
import androidx.appcompat.app.AppCompatActivity


internal object StatusBarColorizer {

    fun setActivityColor(activity: AppCompatActivity) {

        if (Build.VERSION.SDK_INT >= Build.VERSION_CODES.LOLLIPOP) {
            activity.window.statusBarColor = ColorHelper.backgroundColorLightened
        }

        if (activity.supportActionBar != null) {
            activity.supportActionBar!!.setBackgroundDrawable(ColorDrawable(ColorHelper.backgroundColor))
        }

    }
}
